src/rules/valid-expect.ts

```diff
@@ -4,7 +4,9 @@
  */
 
 import { AST_NODE_TYPES, type TSESTree } from '@typescript-eslint/utils';
+import type { RuleFix } from '@typescript-eslint/utils/ts-eslint';
```

Comment:
can you pull this type from `TSESTree` (or it might be `TSESLint` - either way, you can access it through the `@typescript-eslint/utils` import)

Reply:
Thanks! Fixed in 3b27d09.